capten/agent/pkg/agent/agent_sync_app.go

```diff
@@ -0,0 +1,149 @@
+package agent
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/gocql/gocql"
+	"github.com/kube-tarian/kad/capten/agent/pkg/agentpb"
+	"gopkg.in/yaml.v2"
+)
+
+func (a Agent) syncApp(ctx context.Context, request *agentpb.SyncAppRequest) error {
+
+	var appConfig AppConfig
+	if err := yaml.Unmarshal(request.Payload, &appConfig); err != nil {
+		a.log.Errorf("could not unmarshal appConfig yaml: %v", err)
+		return err
+	}
+
+	// Note: We are creating keyspace and table when needed
+	if err := a.store.Session().Query(fmt.Sprintf(createKeyspaceCQL, "1")).Exec(); err != nil {
+		a.log.Errorf("could not create keyspace, err: %v", err)
+		return err
+	}
+
+	if err := a.store.Session().Query(createTableQuery).Exec(); err != nil {
+		a.log.Errorf("could not create table, err: %v", err)
+		return err
+	}
```

Comment:
this need to be created once, not with every API call.
it need to be created during startup if not exist.

Reply:
also migration support needed in case table modified.
casaandra DB package supports db-migration

Reply:
done